tests/phpunit/CRM/Contribute/ActionMapping/ByTypeTest.php

```diff
@@ -354,6 +368,21 @@ public function testTokenRendering(): void {
       'contribution_status_id:label',
     ];
     $processor = new CRM_Contribute_Tokens();
+    $legacyTokens = [];
+    $realLegacyTokens = [];
+    foreach (CRM_Core_SelectValues::contributionTokens() as $token => $label) {
+      $legacyTokens[substr($token, 14, -1)] = $label;
+      if (strpos($token, ':') === FALSE) {
+        $realLegacyTokens[substr($token, 14, -1)] = $label;
+      }
+    }
+    $fields = (array) Contribution::getFields()->addSelect('name', 'title')->execute()->indexBy('name');
+    $allFields = [];
+    foreach ($fields as $field) {
+      $allFields[$field['name']] = $field['title'];
+    }
+    // $this->assertEquals($realLegacyTokens, $allFields);
```

Comment:
This is commented out because it is not true yet & I want to discuss whether it should be TRUE